### PR TITLE
Assign ConfigMap tests to SIG node.

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

File: test/test_owners.csv [hunk 1]
```diff
@@ -23,20 +23,20 @@ Cluster size autoscaling should increase cluster size if pods are pending due to
...
+ConfigMap optional updates should be reflected in volume,timothysc,1,node
+ConfigMap should be consumable from pods in volume,alex-mohr,1,node
+ConfigMap should be consumable from pods in volume as non-root,rrati,0,node
+ConfigMap should be consumable from pods in volume as non-root with FSGroup,roberthbailey,1,node
+ConfigMap should be consumable from pods in volume as non-root with defaultMode and fsGroup set,rrati,0,node
+ConfigMap should be consumable from pods in volume with defaultMode set,Random-Liu,1,node
+ConfigMap should be consumable from pods in volume with mappings,rrati,0,node
+ConfigMap should be consumable from pods in volume with mappings and Item mode set,eparis,1,node
+ConfigMap should be consumable from pods in volume with mappings as non-root,apelisse,1,node
+ConfigMap should be consumable from pods in volume with mappings as non-root with FSGroup,zmerlynn,1nodeapps
```
Comment (dchen1107) [inline test/test_owners.csv]: s/nodeapps/node?

**Comment by dchen1107** [comment]
@erictune My first impression is configmap related failures should be handled by @kubernetes/sig-storage-bugs. But given the recent failures are caused by cgroup cleanup introduced by @kubernetes/sig-node-bugs. I am ok with this change. 

/lgtm

**Comment by dchen1107** [comment]
Please fix above nit, and we are ready to go.

/approve

File: test/test_owners.csv (hunk 1 above)
Comment (gmarek) [inline test/test_owners.csv]: @erictune

**Pull request closed without merging**